Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -895,10 +895,16 @@ endOfLine :
     | whiteSpace? commentOrAnnotation
 ;
 
-// we expect endOfStatement to consume all trailing whitespace
+// We expect endOfStatement to consume all trailing whitespace.
```

Comment:
could we add an "including empty lines" here to make it clearer? I needed a bit to figure out what the change does

Reply:
The only difference to before the change is that there is one hierarchy level more in order to make it easier to make it easier to get the individual statement terminators.